### PR TITLE
Add docs on custom OAuth 2 Exceptions

### PR DESCRIPTION
**Pull request by wilsonge**
Some sort of docs. They really aren't my strong point xD

### CODE REVIEW

File: docs/exceptions.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Exception Handling
+==================
+Rendering Exceptions through the Flask Handler
+----------------------------------------------
+Flask by default contains an exception handler, which connexion's app can proxy
+to with the ``add_error_handler`` method. You can hook either on status code's
```
Comment (hjacobs) [inline docs/exceptions.rst]: "status codes"

File: docs/exceptions.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+exception and render it in some sort of custom format. For example
+
+
+.. code-block:: python
+
+    import connexion
+    from connexion.exceptions import OAuthResponseProblem
+
+    def render_unauthorized(exception):
+        return Response(response=json.dumps({'error': 'There is an in the oAuth token supplied'}), status=401, mimetype="application/json")
```
Comment (hjacobs) [inline docs/exceptions.rst]: error message is missing a word?

**Comment by hjacobs** [comment]
@wilsonge thank you for your contributions!

**Comment by hjacobs** [comment]
:+1:

**Comment by rafaelcaricio** [comment]
@wilsonge Thank you!

**Comment by wilsonge** [comment]
Your welcome :)

**Pull request merged**